### ISSUE TITLE
Add term transverse terminal lower limb defect

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10480]
**Preferred term label:**
Transverse terminal lower limb defect

**Synonyms**
Transverse terminal lower limb reduction

**Definition (free text, please give PubMed ID)**
A congenital anomaly that appears as an “amputation” at different levels of the leg, e.g. femur, foreleg, ankle or toe. The limb is missing the terminal (distal) segment(s), with preservation of all the segment(s) proximal to the missing segment.
PMID: 34263545

**Parent term (use hpo.jax.org/app)**
Transverse terminal limb defect #10479 

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Adams-Oliver syndrome (OMIM100300)

**Your nano-attribution (ORCID)**
A. Zankl 0000-0001-8612-1062, @azankl
L. Wit 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000946
- label: Transverse terminal lower limb defect
- parents: [Transverse terminal limb defect (HP:6000818)](https://hpo.jax.org/app/browse/term/HP:6000818)
- pmid: PMID:34263545
- def: A congenital anomaly that appears as an amputation at different levels of the leg, e.g. femur, foreleg, ankle or toe. The limb is missing the terminal (distal) segment(s), with preservation of all the segment(s) proximal to the missing segment.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10480
- synonyms:
